src/main/java/com/endava/soa/pageObject/HomePage.java

```diff
@@ -26,5 +38,92 @@
     private List<WebElement> anunturiList;
     @FindBy(xpath = "//div[@id='js-recommended-container']")
     private WebElement recomandateContainer;
+    @FindBy(xpath = "//*[@id='m__user_panel']/div")
+    private WebElement cabinetPersonal;
+
+    public WebElement cabinetPersonal() {
+        return (new WebDriverWait(driver, 10))
+                .until(ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id='m__user_panel']/div")));
```

Comment:
.until(ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id='m__user_panel']/div")));
 I suggest to use .until(ExpectedConditions.presenceOfElementLocated(cabinetPersonal));

Reply:
tryed and not working:(